### PR TITLE
Update leaderboards for 2021

### PR DESCRIPTION
**Pull request by spfrantz**
I have added this year's new hashtagged pointless prizes and removed a few that are 1) not sponsored and 2) not attracting participation this year. Nearly all of the new prizes can be accomplished with the /pointless/hashtag view. Civil War History has a more complicated scoring system and so gets its own route and template.

### CODE REVIEW

**Review by merlinorg — DISMISSED** [review]
LGTM. @obscurerichard just moved the parkfinder leaderboard to generic in #204 so you are undoing that, but I don't know what the original motivation was.

File: freezing/web/templates/base.html [hunk 1]
```diff
@@ -120,31 +120,30 @@
...
                             <li><a href="/pointless/arlington">Loopy for Arlington</a></li>
-                            <li><a href="/pointless/generic/lostdog">Lost Dog Rides</a></li>
-                            <li><a href="/pointless/hashtag/bafsmetro">METRO PEAK PIONEER Rides</a></li>
                             <li><a href="/pointless/opmdays">Michigander Snow Day Rides</a></li>
-                            <li><a href="/pointless/generic/parkfinder">Park Finder</a></li>
+                            <li><a href="/pointless/hashtag/parkfinder">Park Finder</a></li>
                             <li><a href="/pointless/points_per_mile">Points per Mile</a></li>
-                            <li><a href="/pointless/generic/poggio-indiv">POGGIO (Indiv)</a></li>
-                            <li><a href="/pointless/generic/poggio-team">POGGIO (Team)</a></li>
-                            <li><a href="/pointless/generic/suppleride">Supple Ride</a></li>
```
Comment (merlinorg) [inline freezing/web/templates/base.html:143]: There are some folks diligently tagging their #supplerides.

**Comment by spfrantz** [comment]
Thanks Merlin -- the ParkFinder leaderboard wasn't working (hashtag needed updating) so treated it the same as the other simple hashtag-based competitions and used the `/hashtag` route. Not sure what people's preferences are for when to use `/hashtag` versus `/generic`.

File: freezing/web/templates/base.html (hunk 1 above)
Comment (spfrantz) [reply freezing/web/templates/base.html:143]: Noted! I have restored that leaderboard.

Comment (obscurerichard) [reply freezing/web/templates/base.html:143]: Really any pointless prizes where people are tagging their rides we should leave up, with a note that there is no prize sponsor announced for the current year.

**Comment by obscurerichard** [comment]
I think this needs more work before we push it live. It cut too deep into removing leaderboards that people use, even if there's no prize sponsor, and we should prefer the Generic leaderboards over the Hashtag ones because you get full control over Generic leaderboards including having multiple hashtags that go into the prize.

Generally we don't like hashtags that have the year embedded in them so we add an alternate hashtag into the generic leaderboards so that we can make them evergreen more easily. We spent time *removing* year-based hash tags both this year and last year. We should socialize that more - don't embed the year or BAFS into your hashtags for pointless prizes.

**Pull request merged**